esmvaltool/recipes/schlund20jgr/recipe_schlund20jgr_gpp_change_1pct.yml

```diff
@@ -0,0 +1,381 @@
+# ESMValTool
+# recipe_schlund20jgr_gpp_change_1pct.yml
+---
+documentation:
+
+  description: |
+    Plot emergent constraint of Wenzel et al. (2016) for global mean gross
+    primary production (GPP) in biogeochemically-coupled 1%CO2 run.
+
+  authors:
+    - schlund_manuel
+
+  maintainer:
+    - schlund_manuel
+
+  references:
+    - schlund20jgr
+    - wenzel16nat
+
+  projects:
+    - 4c
+    - crescendo
+    - usmile
+
+
+# YAML anchors
+
+PROVENANCE_ANCHOR: &provenance
+  domains:
+    - global
```

Comment:
```suggestion
```
This doesn't work, domains can only be specified in the provenance coming from the diagnostic, see also my earlier comment: https://github.com/ESMValGroup/ESMValTool/pull/1860#pullrequestreview-510390248. Probably best to remove it everywhere, since it doesn't work anyway if you add it like this.